.github/workflows/prerelease.yml

```diff
@@ -42,7 +42,7 @@ jobs:
             -DSTRINGZILLA_BUILD_BENCHMARK=1 \
             -DSTRINGZILLA_BUILD_TEST=1
 
-          cmake --build build_artifacts --config RelWithDebInfo > build_artifacts/logs.txt 2>&1 || {
+          cmake --build build_artifacts --config RelWithDebInfo > build_artifacts\logs.txt 2>&1 || {
```

Comment:
Isn't this supposed to be a normal slash?

Reply:
It doesn't really matter (either is acceptable in this use case), but back slashes were needed for the `type` command, so I just changed them all.